package.json

```diff
@@ -22,9 +22,9 @@
     "@react-native-async-storage/async-storage": "^1.17.7",
     "@react-native-firebase/app": "^16.5.0",
     "@react-native-firebase/messaging": "^16.5.0",
-    "@react-navigation/bottom-tabs": "^6.x.x",
-    "@react-navigation/native": "^6.x.x",
-    "@react-navigation/stack": "^6.x.x",
+    "@react-navigation/bottom-tabs": "6.5.8",
+    "@react-navigation/native": "6.1.7",
+    "@react-navigation/stack": "6.3.17",
```

Comment:
Oh, it is lacking the update in the package-lock.json as well.

Reply:
I actually used the versions that were in the `package-lock.json`, so it wasn't changed.

Also, I removed the `^` operator, so that we lock this version for more supply chain security, as was discussed on this project.

Reply:
:dart: Thanks for pointing that out! The `package-lock` was indeed incorrect.
Fixed it on [af4d4d7](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/306/commits/af4d4d7fb9402befee1972ad29df0c741d9df099)